### PR TITLE
Merge pull request #32 from zapier/bump-preset-env-version

### PR DESCRIPTION
**Pull request by msholty-fd**
# Description

This PR bumps `@babel/preset-env` so we can start using a couple really awesome new features in zapier/zapier:

## Optional Chaining

[Quoting from this blog](https://2ality.com/2019/07/optional-chaining.html)

The following kinds of optional operations exist.
```js
obj?.prop     // optional static property access
obj?.[«expr»] // optional dynamic property access
func?.(«arg0», «arg1») // optional function or method call
```
The rough idea is:

If the value before the question mark is neither undefined nor null, then perform the operation after the question mark.
Otherwise, return undefined.

## Nullish coalescing

[Quoting from this blog](https://2ality.com/2019/08/nullish-coalescing.html)

The result of actualValue ?? defaultValue is:

defaultValue if actualValue is undefined or null
actualValue otherwise
Examples:
```js
> undefined ?? 'default'
'default'
> null ?? 'default'
'default'

> false ?? 'default'
false
> '' ?? 'default'
''
> 0 ?? 'default'
0
```

## Steps to test this out!

1. In the repo on your machine, run `yarn link`
2. Copy and paste what it returns to you in `zapier/zapier`: `yarn link "@zapier/babel-preset-zapier"`
3. Here's the tricky part: This branch introduces some new dependency changes that will not immediately reflect in `zapier/zapier` once you've yarn link'd. You'll need to explicitly install `@babel/preset-env@7.8.3` to simulate what will happen when we install this package from `npm`.
4. Try out some new JS syntax, run `yarn @zapier/web dev` and see that it works!

## For more info...

If you want to know more about some of the new features we'll have, please check out [this blog post](https://2ality.com/2019/12/ecmascript-2020.html)

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -59,6 +59,29 @@ describe('babel-preset-zapier', () => {
...
+  });
+
+  it('parses nullish coalescing for JS', () => {
+    const code = `
+      const foo = undefined ?? 'default';
+    `;
+
+    expect(transform(code)).toMatchSnapshot();
+  });
+
```
Comment (vitorbal) [inline index.test.js]: Thanks for the tests!

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
```
Comment (msholty-fd) [inline package.json]: @vitorbal @ngryman I just thought of this, but with the large leap in versions of `preset-env`, we may be able to remove these explicit dependencies for specific plugins in favor of just using whatever `preset-env` depends on.

Comment (vitorbal) [reply package.json]: @msholty-fd makes sense! Do you know if `plugin-transform-classes` is also supported by `preset-env`?

**Comment by jdaudier** [comment]
@msholty-fd Now that this is merged, is there anything I need to do to use these new features in z/z?

File: package.json (hunk 2 above)
Comment (msholty-fd) [reply package.json]: @vitorbal I do not! We can always check when we try it out in `zapier/zapier`

**Comment by msholty-fd** [comment]
@jdaudier Yes, we need to publish this release version and update `z/z` to use that version.

**Comment by jdaudier** [comment]
@msholty-fd Ok, I'll stay tuned for your grand announcement! 🎉

**Pull request merged**